share/tools/create_manpage_completions.py

```diff
@@ -873,8 +873,11 @@ def get_paths_from_manpath():
         manpath, err_data = proc.communicate()
     parent_paths = manpath.decode().strip().split(':')
     if not parent_paths:
-        sys.stderr.write("Unable to get the manpath (tried `%s`)\n" % " ".join(program))
-        sys.exit(-1)
+        # HACK: Use some fallback in case we can't get anything else.
+        # `mandoc` does not provide `manpath` or `man --path` and $MANPATH might not be set, so just use he default for mandoc (minus /usr/X11R6/man, because that's not relevant).
```

Comment:
s/use he default/use the default/

Also, I verified that on macOS adding an invalid directory to manpath causes no problems.